src/app/fdctl/run/tiles/quic.c

```diff
@@ -31,6 +32,12 @@ init( fd_tile_args_t * args ) {
     FD_LOG_ERR(( "OPENSSL_init_ssl failed" ));
   if( FD_UNLIKELY( !OPENSSL_init_crypto( OPENSSL_INIT_LOAD_CRYPTO_STRINGS | OPENSSL_INIT_NO_LOAD_CONFIG , NULL ) ) )
     FD_LOG_ERR(( "OPENSSL_init_crypto failed" ));
+
+  /* ensure netlink is initialized before sandboxing,
+     and set fd to 4 */
+  if( fd_nl_set_fd( 4 ) ) {
```

Comment:
I'm not sure that you need to be this specific, you should just be able to do `fd_ip_join` in this privileged part? Eg, see the shred PR at https://github.com/firedancer-io/firedancer/pull/753/files#diff-6a7d3ef4e5e1c3fcfbfeb0b4967c38c839cff1cc3562bedca73fb222200869deR15

Then you can remove this set_fd piece